repository/src/main/java/by/academy/jee/util/Initializer.java

```diff
@@ -0,0 +1,38 @@
+package by.academy.jee.util;
+
+import by.academy.jee.dao.person.PersonDao;
+import by.academy.jee.dao.person.admin.AdminDaoForMemoryDatabase;
+import by.academy.jee.dao.person.teacher.TeacherDaoForMemoryDatabase;
+import by.academy.jee.database.Database;
+import by.academy.jee.model.person.Admin;
+import by.academy.jee.model.person.Teacher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public abstract class Initializer {
+
+    private static final Logger log = LoggerFactory.getLogger(Initializer.class);
+
+    public static PersonDao<Admin> adminDao;
+    public static PersonDao<Teacher> teacherDao;
```

Comment:
Нарушение инкапусляции. Все поля должны быть приватными с методами доступа к ним.
```suggestion
    private static PersonDao<Admin> adminDao;
    private static PersonDao<Teacher> teacherDao;
```

Reply:
Done